### PR TITLE
Added missing input and fixed other input types structures in Pet Insuarence 

### PR DESCRIPTION
**Pull request by Xedros**

### CODE REVIEW

File: schema/pet-insurance.json [hunk 1]
```diff
@@ -592,5 +569,4 @@
             "description": "Breed type is required by the website"
         }
     ]
-
 }
```
Comment (inca) [inline schema/pet-insurance.json]: !

File: schema/pet-insurance.json [hunk 2]
```diff
@@ -505,12 +487,8 @@
...
                         }
                     }
-                }
-            },
-
-            "Address": {
-                "type": "object",
-                "properties": {
+                },
+                "address": {
```
Comment (inca) [inline schema/pet-insurance.json]: We have `Generic.Address` — why don't we use it here?

File: schema/pet-insurance.json [hunk 3]
```diff
@@ -288,25 +289,20 @@
...
             "maxItems": 3,
             "items": {
-                "$ref": "#/domains/PetInsurance/types/VoluntaryExcessItem"
+                "$ref": "#/domains/PetInsurance/types/VoluntaryExcess"
             }
         },
-        "VoluntaryExcessItem": {
+        "VoluntaryExcess": {
             "type": "object",
             "description": "Flight fare information.",
```
Comment (inca) [inline schema/pet-insurance.json]: This (and a number of other definitions which mention passengers) are most likely a copy-paste leftovers. Please fix

File: schema/pet-insurance.json (hunk 1 above)
Comment (Xedros) [reply schema/pet-insurance.json]: That's not an inserted hyphen. It represent a deleted empty line.
image

Comment (inca) [reply schema/pet-insurance.json]: I undertand that :) Configure your editor to emit a single blank line when saving a file.

File: schema/pet-insurance.json (hunk 3 above)
Comment (Xedros) [reply schema/pet-insurance.json]: image

I don't understand. What should I modify? Those red-lines are the lines before modification. They are not in code anymore.

Comment (Xedros) [reply schema/pet-insurance.json]: Do you mean the description?

File: schema/pet-insurance.json (hunk 2 above)
Comment (Xedros) [reply schema/pet-insurance.json]: Because there are elements not needed (like: City, Country)

File: schema/pet-insurance.json (hunk 3 above)
Comment (inca) [reply schema/pet-insurance.json]: Yes, "flight fare information". Also search the same number for "passenger" text.

File: schema/pet-insurance.json (hunk 2 above)
Comment (inca) [reply schema/pet-insurance.json]: Ok, this seems like a broader topic (fyi we had a discussion about addresses last week). Feel free to merge this one for now.

Comment (randunel) [reply schema/pet-insurance.json]: @Xedros the domain must be as universal as possible. I'm pretty sure you'll find a pet insurance provider requiring a more complete address.
This domain must cater to as many future pet insurance providers as possible, not just the one you've scripted.

File: schema/pet-insurance.json [hunk 4]
```diff
@@ -78,7 +77,11 @@
             "typeRef": "#/domains/PetInsurance/types/CoverOptions",
             "sourceOutputKey": "availableCoverOptions",
             "inputMethod": "SelectMany"
-
+        },
+        "selectedVetPaymentTerm": {
```
Comment (randunel) [inline schema/pet-insurance.json]: needs `description`, what is it and how is it different from `selectedPaymentTerm`?

File: schema/pet-insurance.json [hunk 5]
```diff
@@ -78,7 +77,11 @@
...
             "sourceOutputKey": "availableCoverOptions",
             "inputMethod": "SelectMany"
-
+        },
+        "selectedVetPaymentTerm": {
+            "typeRef": "#/domains/PetInsurance/types/VetPaymentTerm",
+            "sourceOutputKey": "availableVetPaymentTerms",
+            "inputMethod": "SelectOne"
         },
         "selectedPaymentTerm": {
```
Comment (randunel) [inline schema/pet-insurance.json]: Needs `description`, how is it different from `selectedVetPaymentTerm`?

File: schema/pet-insurance.json [hunk 6]
```diff
@@ -115,9 +118,11 @@
         "availableCoverOptions": {
             "typeRef": "#/domains/PetInsurance/types/CoverOptions"
         },
+        "availableVetPaymentTerms": {
```
Comment (randunel) [inline schema/pet-insurance.json]: needs `description`, how is it different from `availablePaymentTerms`?

File: schema/pet-insurance.json [hunk 7]
```diff
@@ -346,6 +336,19 @@
                 "price"
             ]
         },
+        "VetPaymentTerm": {
```
Comment (randunel) [inline schema/pet-insurance.json]: Can `PaymentTerm` be used instead? They look the same to me. Elimitate duplicate (identical) definitions, please, and add the descriptions to inputs/outputs instead.

File: schema/pet-insurance.json (hunk 1 above)
Comment (randunel) [reply schema/pet-insurance.json]: Always end the lines with newline, misconfigured editor.

File: schema/pet-insurance.json [hunk 8]
```diff
@@ -288,25 +284,20 @@
             "minItems": 1,
             "maxItems": 3,
             "items": {
-                "$ref": "#/domains/PetInsurance/types/VoluntaryExcessItem"
+                "$ref": "#/domains/PetInsurance/types/VoluntaryExcess"
```
Comment (randunel) [inline schema/pet-insurance.json]: Why is `maxItems` 3? Can't there be 4 voluntary excess options on a different website?

**Pull request merged**